src/Todo/Footer.purs

```diff
@@ -33,12 +32,15 @@ instance writeVisibility :: WriteForeign Visibility where
 
 type Props =
   { tasks            :: Array Task
-  , onClearCompleted :: Effect Unit
+  , onClearCompleted :: Events.EventHandler
   , visibility       :: Visibility
   }
 
-component :: React.Component Props
-component = React.stateless { displayName: "Footer", render }
+comp :: React.Component Props
+comp = React.createComponent "Footer"
+
+component :: Props -> JSX
+component = React.makeStateless comp render
```

Comment:
Do we have a use for both `comp` and `component` or we can absorb `comp` in here?

Reply:
IIUC the note here: 

https://pursuit.purescript.org/packages/purescript-react-basic/6.2.0/docs/React.Basic#v:createComponent

then createComponent should always be created/specified at the module level. I had thought about doing `makeStateless (createComponent "Footer") render`, but that note made me hesitate 

Should I rename these, though? Do you think I understood the docs correctly?

Reply:
Oh right, I missed these docs, thanks 👍 

Let's rename them for clarity:
- `comp` → `component`
- `component` → `footer`